### DISCUSSION TITLE
One to Many relationship displays ID and not name in Data Provider Table

### DISCUSSION THREAD

**Discussion by sneakersgames** [Budibase/budibase#10841] [Help] [answered]
Hello,

I have a One (club) to Many (players) relationship. In the Data view, I can correctly see the name of club the player belongs to. However, when I use a Data Provider Table component, it does not display the name of the club but the ID of the club row.

![image](https://github.com/Budibase/budibase/assets/473894/28040556-5add-44b2-a0fe-467741875223)

As you can see, on the club Data Provider Table, the name of the players are correctly listed. How do I get club name and not club_id on the players Table.

Thanks!

**Comment by sneakersgames** [comment]
Anyone? 😄

**Reply by ducnhoxlatui** [reply, marked as answer]
Hi @sneakersgames , i think that you can go to Data screen. As the club table, edit the column that you want to show to another table as Display by check the "Use as table display column"

Example image:
![image](https://github.com/Budibase/budibase/assets/71976420/848fae27-6fd5-48ea-95ca-9b8fcd2e1d32)

Hope this will help you

**Reply by melohagan** [reply]
This is spot on @ducnhoxlatui thank you!  

https://docs.budibase.com/docs/budibasedb#editing-columns
